paddle/fluid/platform/mkldnn_reuse.h

```diff
@@ -72,6 +73,17 @@ class MKLDNNHandlerT {
     return backward_p;
   }
 
+  std::shared_ptr<TBackward_params> AcquireBackwardWeightsPrimitive() {
+    const std::string key_p = key_ + "@bwd_w_p";
```

Comment:
Shouldn't we have those suffixes like `"@bwd_w_p"`  in some form of enums?

Reply:
Yes we should . What we actully need is suffixes as enum and then mapping mechanism "key  -> string" to have it printed as part of VLOG.